solidity/test/relay/LightRelayMaintainerProxy.test.ts

```diff
@@ -0,0 +1,532 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+
+import { ethers, deployments, helpers, waffle } from "hardhat"
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers"
+import { expect } from "chai"
+import { ContractTransaction, BigNumber } from "ethers"
+import type {
+  LightRelay,
+  LightRelayMaintainerProxy,
+  ReimbursementPool,
+} from "../../typechain"
+import { concatenateHexStrings } from "../helpers/contract-test-helpers"
+import longHeaders from "./longHeaders.json"
+
+const { provider } = waffle
+
+const { createSnapshot, restoreSnapshot } = helpers.snapshot
+
+const ZERO_ADDRESS = ethers.constants.AddressZero
+
+const fixture = async () => {
+  await deployments.fixture()
+
+  const { deployer, governance } = await helpers.signers.getNamedSigners()
+  const [thirdParty, maintainer] = await helpers.signers.getUnnamedSigners()
+
+  const reimbursementPool: ReimbursementPool =
+    await helpers.contracts.getContract("ReimbursementPool")
+
+  const lightRelayMaintainerProxy: LightRelayMaintainerProxy =
+    await helpers.contracts.getContract("LightRelayMaintainerProxy")
+
+  const lightRelay: LightRelay = await helpers.contracts.getContract(
+    "LightRelay"
+  )
+
+  await lightRelay.connect(deployer).setAuthorizationStatus(true)
+
+  return {
+    deployer,
+    governance,
+    maintainer,
+    thirdParty,
+    reimbursementPool,
+    lightRelayMaintainerProxy,
+    lightRelay,
+  }
+}
+
+describe("LightRelayMaintainerProxy", () => {
+  let deployer: SignerWithAddress
+  let governance: SignerWithAddress
+  let maintainer: SignerWithAddress
+  let thirdParty: SignerWithAddress
+  let reimbursementPool: ReimbursementPool
+  let lightRelayMaintainerProxy: LightRelayMaintainerProxy
+  let lightRelay: LightRelay
+
+  before(async () => {
+    // eslint-disable-next-line @typescript-eslint/no-extra-semi
+    ;({
+      deployer,
+      governance,
+      maintainer,
+      thirdParty,
+      reimbursementPool,
+      lightRelayMaintainerProxy,
+      lightRelay,
+    } = await waffle.loadFixture(fixture))
+
+    await deployer.sendTransaction({
+      to: reimbursementPool.address,
+      value: ethers.utils.parseEther("100"),
+    })
+  })
+
+  describe("authorize", () => {
+    context("When called by non-owner", () => {
```

Comment:
Nitpicking: let's lowercase:

```suggestion
    context("when called by non-owner", () => {
```

Reply:
Done in 462925b1bd07e05f87beb7c9fcba541d335c0be1.